stko/molecular/molecule_splitter/molecule_splitter.py

```diff
@@ -0,0 +1,157 @@
+"""
+Molecule Splitter
+=================
+
+#. :class:`.MoleculeSplitter`
+
+Class for splitting a molecule into many with new connectors.
+
+"""
+
+import logging
+from rdkit.Chem import AllChem as rdkit
+from itertools import combinations
+
+import stk
+
+logger = logging.getLogger(__name__)
+
+
+class MoleculeSplitter:
+    """
+    Split an stk.molecule into many with new functional groups.
+
+    Examples
+    --------
+
+    Given a molecule, this class allows you to break bonds based on
+    `breaker_smarts` between the atoms in `bond_deleter_ids` and cap
+    the broken bond by an atom defined in `replacer_smarts`.
+
+    .. code-block:: python
+
+        import stk
+        import stko
+
+        full_mol = stk.BuildingBlock('C1=CC=NC(=C1)C=NC2=CC=C(C=C2)Br')
+
+        splitter = stko.MoleculeSplitter(
+            breaker_smarts='[#6X3]~[#7X2]~[#6X3H1]~[#6X3!H1]',
+            bond_deleter_ids=(0, 1),
+            replacer_smarts='[Br]',
+            functional_groups=(stk.BromoFactory(), ),
```

Comment:
Unnecessary space here?

Reply:
Not that I can tell. It should be a space to fit the formatting of an iterable (tuple).